src/js/example-app.html

```diff
@@ -6,6 +6,13 @@
   <script src="https://unpkg.com/react-dom@16/umd/react-dom.development.js" crossorigin></script>
   <!-- this HTML snippet is appended to <<imports>> -->
   <script src="https://unpkg.com/babel-standalone@6/babel.min.js"></script>
+  <!-- this HTML snippet is appended to <<imports>>  -->
+  <script src="https://unpkg.com/@rjsf/core/dist/react-jsonschema-form.js"></script>
+  <!-- this HTML snippet is appended to <<imports>>  -->
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha384-HSMxcRTRxnN+Bdg0JdbxYKrThecOKuH5zCYotlSAcp1+c8xmyTe9GYg1l9a69psu" crossorigin="anonymous">
+  <script src="https://cdn.jsdelivr.net/npm/vega@5.12.1"></script>
```

Comment:
Vega 5.13 has a lot of bug fixes (mostly related to SVG etc.). It would be nice to update this as well.

Reply:
Done